var/spack/repos/builtin/packages/py-iterative-stats/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIterativeStats(PythonPackage):
+    """Bacis iterative statistics implementation."""
+
+    pypi = "iterative-stats/iterative-stats-4.0.0.zip"
+    git = "https://github.com/IterativeStatistics/BasicIterativeStatistics.git"
+    maintainers("frobinou")
```

Comment:
@frobinou do you consent to maintaining this build recipe?

Reply:
If you don't get explicit permission you could just remove this

Reply:
I know @Frobinou to be very busy these days but she gave me her agreement two weeks ago when I informed her I was making a Spack package for her library.